www/code/Makefile.am

```diff
@@ -1,5 +1,27 @@
 include @top_srcdir@/include.mk
-include @top_srcdir@/examples.mk
+
+# For reproducible builds. Neither of these are used.
```

Comment:
What does "Neither of these are used" refer to here?

Reply:
That passage means that these changes are made for reproducible
builds and that none of the variables/programs are used during the
build of yaws. At least so far as I could tell.

The only thing that happens is that Makefile.in and Makefile are
generated when running autoconf and configure in the top level
yaws source dir.

It doesn't seem to be possible to build www/code or www/shoppingcart
out of tree (e.g. from the installed path) because it is missing the
include of ../../erlang_deps.mk in www/code/Makefile.

These should probably be fixed if they are supposed to be built.
They currently can not be built in the source tree or (installed)
out of tree. They do however break the reproducible builds.

Reply:
I see, thanks for the explanation. I'll amend your commit slightly to clarify these comments, then merge. Thanks for fixing this.

Reply:
Thanks!